docs/logs/pipelines.md

```diff
@@ -0,0 +1,82 @@
+title: Log Pipelines
+description: Add processing steps to log ingestion
+
+Sometimes raw log events are not exactly the way you want them to be and it might be hard to set up log shipper that is able to transform or filter log events. An alternative approach is to configure processing steps that are injected in ingestion flow. We call that pipelines. Pipelines can be used to drop unwanted log events, remove unwanted fields, enrich or transform your documents.
+Pipelines are easily accessible and link to pipeline builder is available in the main menu of every log report.
+
+![Pipelines Button](../../images/logs/pipelines/pipeline-button.png)
+
+
+### Pipeline Builder
+Pipeline builder is opened as a full page and has basic structure as any other page.
+
+![Pipeline Builder](../../images/logs/pipelines/pipeline-builder-saved.png)
+
+On top we have header and subnav that can be used to pick documents for preview. The page is split between resizable preview section and pipeline configuration section. Preview shows documents before and after processing. Pipeline configuration section has list of processors on the left and configuration details for a selected processor on the right.
+
+Header and subnav let you define time and filters that will be used to load sample data for previewing pipeline.
+
+![Pipeline Builder](../../images/logs/pipelines/pipeline-builder-header.png)
+
+By default, we will load only 10 documents but you can also change it to 50 or 100. Note that larger number of documents may slow preview a bit down since documents are sent for processing on each pipeline configuration change.
+
+#### Processors
+Processors are units of processing in pipelines. They can change, drop or even produce additional events. Their are chained to form a pipeline. The output of one processor is the input for the next processor.
+Configuration section will be empty in most cases when you initially open pipeline builder.
+
+![Empty Pipeline](../../images/logs/pipelines/empty-pipeline.png)
+
+In such cases, instead of selected processor configuration, there will be list of available processors. Click on any processor card will add processor with default values. Processors can also be added by clicking on "Add Processor" button.
+
+![Add Processor](../../images/logs/pipelines/add-processor.png)
+
+When clicked on the button, a modal with a searchable list of processors will appear to select processor to add.
+You can also add processor by duplicating some of existing processors.
+
+![Processor Actions](../../images/logs/pipelines/processor-actions.png)
+
+Duplicate action is under our standard action button in the list of processors. Note that order of processors matter since the output of one processor is the input for the next processor. You can reorder processor using drag handle left from the processor's name.
```

Comment:
```suggestion
The duplicate action is below the standard action button in the list of processors. Note that **the order of processors matters** since the **output of one processor is the input for the next processor**. You can reorder processors using the drag handle to the left of the processor's name.
```